geode-redis/src/main/java/org/apache/geode/redis/internal/executor/key/RenameExecutor.java

```diff
@@ -16,34 +16,35 @@
 
 package org.apache.geode.redis.internal.executor.key;
 
-import static org.apache.geode.redis.internal.RedisConstants.ERROR_NO_SUCH_KEY;
-
 import java.util.List;
 
+import org.apache.geode.redis.internal.RedisConstants;
 import org.apache.geode.redis.internal.data.ByteArrayWrapper;
 import org.apache.geode.redis.internal.executor.AbstractExecutor;
-import org.apache.geode.redis.internal.executor.RedisResponse;
+import org.apache.geode.redis.internal.netty.Coder;
 import org.apache.geode.redis.internal.netty.Command;
 import org.apache.geode.redis.internal.netty.ExecutionHandlerContext;
 
 public class RenameExecutor extends AbstractExecutor {
-
   @Override
-  public RedisResponse executeCommand(Command command,
-      ExecutionHandlerContext context) {
+  public void executeCommand(Command command, ExecutionHandlerContext context) {
```

Comment:
The changes here look like an incorrect merge perhaps.

Reply:
looking into it-  thx